oojs.js

```diff
@@ -1,3 +1,24 @@
+class Playlist {
+  constructor(title, ){
```

Comment:
You forgot to add another argument. Because of the hanging comma, this will error out.